tests/cupyx_tests/scipy_tests/fftpack_tests/test_fftpack.py

```diff
@@ -106,6 +106,31 @@ def test_ifft2_overwrite(self, xp, scp, dtype):
         return scp.fftpack.ifft2(x, shape=self.s, axes=self.axes,
                                  overwrite_x=True)
 
+    @testing.for_complex_dtypes()
+    # TODO: what decorator should we put here? In scipy.fftpack there is no planning,
```

Comment:
I think you could just test the output with an explicit `plan` to the output with automatic planning. The automatic planning case has already been validated vs. scipy, so I think that should be sufficient.

Reply:
OK. I now use `isinstance`  to tell whether the FFT functions are coming from `cupyx.scipy` or just `scipy`, and then generate the output. Not 100% confident about this, though. Let's see if we can pass the CI.